receiver/hostmetricsreceiver/metadata.yaml

```
@@ -50,3 +71,19 @@ metrics:
     unit: 1
     data:
       type: double gauge
+
+  system.filesystem.usage:
+    description: Filesystem bytes used.
+    unit: By
+    data:
+      type: int sum
+      aggregation: cumulative
+      monotonic: false
+
+  system.filesystem.inodes.usage:
+    description: FileSystem iNodes used.
+    unit: "{inodes}"
```

I don't really know what the right unit is here but I've never seen one with braces.

```suggestion
    unit: "1"
```

---

Actually, this is what we currently have. We have the same pattern in other places as well.

- [disk metrics](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.18.0/receiver/hostmetricsreceiver/internal/scraper/diskscraper/disk_metadata.go)
- [network metrics](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.18.0/receiver/hostmetricsreceiver/internal/scraper/networkscraper/network_metadata.go)
- [paging metrics](https://github.com/open-telemetry/opentelemetry-collector/blob/v0.18.0/receiver/hostmetricsreceiver/internal/scraper/pagingscraper/paging_metadata.go)

@jrcamp Would you be ok if we address it separately and stick to what we currently have for now?